packages/nimble-components/package.json

```diff
@@ -26,14 +26,14 @@
     "generate-scss:run": "node build/generate-scss/dist/index.js",
     "tdd": "npm run build-components && npm run test-chrome",
     "tdd:watch": "npm run build-components:watch & npm run test-chrome:watch",
-    "test-chrome:debugger": "karma start karma.conf.js --browsers=ChromeDebugging",
-    "test-chrome:verbose": "karma start karma.conf.verbose.js --browsers=ChromeHeadlessOpt --single-run",
-    "test-chrome:watch": "karma start karma.conf.js --browsers=ChromeHeadlessOpt --watch-extensions js",
-    "test-chrome": "karma start karma.conf.js --browsers=ChromeHeadlessOpt --single-run",
-    "test-firefox": "karma start karma.conf.js --browsers=FirefoxHeadless --single-run",
-    "test-webkit": "karma start karma.conf.js --browsers=WebkitHeadless --single-run",
-    "test-webkit:debugger": "karma start karma.conf.js --browsers=Webkit",
-    "test": "npm run test-chrome:verbose"
+    "test-chrome:debugger": "karma start karma.conf.js --browsers=ChromeDebugging --skip-tags SkipChrome",
+    "test-chrome:verbose": "karma start karma.conf.verbose.js --browsers=ChromeHeadlessOpt --single-run --skip-tags SkipChrome",
+    "test-chrome:watch": "karma start karma.conf.js --browsers=ChromeHeadlessOpt  --skip-tags SkipChrome --watch-extensions js",
+    "test-chrome": "karma start karma.conf.js --browsers=ChromeHeadlessOpt --single-run --skip-tags SkipChrome",
+    "test-firefox": "karma start karma.conf.js --browsers=FirefoxHeadless --single-run --skip-tags SkipFirefox",
```

Comment:
Could you try to make the firefox and webkit configurations match the chrome ones? This includes:
- adding `:verbose` config and calling that from the `test` command. This gives us more detailed info about which tests ran in the build log. Right now I can only see the test count for the firefox run.
- adding `:debugger` and `:watch` configurations for Firefox and Webkit if that's supported
- adding `tdd-firefox`, `tdd-webkit`, `tdd-firefox:watch`, and `tdd-webkit:watch` if that's supported

Reply:
Done. The Chrome browser configs we use add a bunch of options to disable features, but I didn't bother to try to optimize the Firefox or Webkit options. I also wasn't sure about whether we needed to set a remote debugging port, like we do for our debug config of Chrome. I don't know if that's something people use, or what those options would be for Firefox and Webkit.